src/app/series/directives/series-templates.component.ts

```diff
@@ -22,4 +67,54 @@ export class SeriesTemplatesComponent implements OnDestroy {
     this.tabSub.unsubscribe();
   }
 
+  hasVersions() {
+    return this.series.versionTemplates.some(f => !f.isDestroy);
+  }
+
+  addVersion() {
+    let draft = new AudioVersionTemplateModel(this.series.doc);
+    draft.set('label', 'Podcast Audio');
+    this.series.versionTemplates.push(draft);
+  }
+
+  removeVersion(version: AudioVersionTemplateModel) {
+    version.isDestroy = true;
+    if (version.isNew) {
+      this.series.versionTemplates.splice(this.series.versionTemplates.indexOf(version), 1);
+      version.unstore();
+    }
+  }
+
+  canAddFile(version: AudioVersionTemplateModel): boolean {
+    return version.fileTemplates.filter(f => !f.isDestroy).length < 10;
```

Comment:
Arbitrary ... same way I determined the min-length validation for title/description fields.  Ticket #86 is to actually find some sane values for validations.